pkg/semrel/github_test.go

```diff
@@ -12,21 +12,26 @@ import (
 
 	"github.com/Masterminds/semver"
 	"github.com/google/go-github/v30/github"
+	"github.com/stretchr/testify/require"
 )
 
 func TestNewGithubRepository(t *testing.T) {
+	require := require.New(t)
+
 	repo, err := NewGitHubRepository(context.TODO(), "", "", "")
-	if repo != nil || err == nil {
-		t.Fatal("invalid initialization")
-	}
+	require.Nil(repo)
+	require.EqualError(err, "invalid slug")
+
 	repo, err = NewGitHubRepository(context.TODO(), "", "owner/test-repo", "token")
-	if repo == nil || err != nil {
-		t.Fatal("invalid initialization")
-	}
+	require.NotNil(repo)
+	require.NoError(err)
+	require.Equal(repo.Owner(), "owner")
+	require.Equal(repo.Repo(), "test-repo")
+
 	repo, err = NewGitHubRepository(context.TODO(), "github.enterprise", "owner/test-repo", "token")
-	if repo.Client.BaseURL.Host != "github.enterprise" || err != nil {
-		t.Fatal("invalid enterprise initialization")
-	}
+	require.NotNil(repo)
+	require.NoError(err)
+	require.Equal("github.enterprise", repo.Client.BaseURL.Host)
```

Comment:
Can we use for `require.Equal` calls as first argument the value we want to test and as second argument the expected value?

Reply:
unfortunately expected is the first argument. https://godoc.org/github.com/stretchr/testify/require#Equal.
Otherwise the errors come out backwards

Reply:
okay 😅 then we should change the other `require.Equal`:
(not sure if I found all)
https://github.com/go-semantic-release/semantic-release/pull/61/files#diff-d57521d9a1326c3f50f3a9e9a23b9ba3R28
https://github.com/go-semantic-release/semantic-release/pull/61/files#diff-d57521d9a1326c3f50f3a9e9a23b9ba3R29
https://github.com/go-semantic-release/semantic-release/pull/61/files#diff-d57521d9a1326c3f50f3a9e9a23b9ba3R122
https://github.com/go-semantic-release/semantic-release/pull/61/files#diff-4079c0756c016e7e4e1f4dce5637e70eR28
https://github.com/go-semantic-release/semantic-release/pull/61/files#diff-4079c0756c016e7e4e1f4dce5637e70eR29
https://github.com/go-semantic-release/semantic-release/pull/61/files#diff-4079c0756c016e7e4e1f4dce5637e70eR124

Reply:
Double checked and also only found 6. Fixed